egs/wsj0-mix/DeepClustering/model.py

```diff
@@ -33,8 +35,31 @@ def make_model_and_optimizer(conf):
     enc = fb.Encoder(fb.STFTFB(**conf['filterbank']))
     masker = ChimeraPP(int(enc.filterbank.n_feats_out/2), 2,
                        embedding_dim=20, n_layers=2, hidden_size=600, \
-                       dropout=0, bidirectional=True)
+                       dropout=0.5, bidirectional=True, \
+                       log=conf['filterbank']['log'])
```

Comment:
These constants should go in `conf.yml`, expose the ones that we'll experiment with in the run.sh as wel.